cpu/nrf52/radio/Kconfig

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2021 Inria
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+config HAVE_NRF52_RADIO
+    bool
+    select MODULE_NRF52_RADIO if MODULE_NETDEV_DEFAULT
+    help
+        Indicates that an NRF52 radio is present.
+
+menuconfig MODULE_NRF52_RADIO
+    bool "NRF52 radio driver"
+    depends on HAVE_NRF52_RADIO
+    depends on TEST_KCONFIG
+
+if MODULE_NRF52_RADIO
+
+choice
+    bool "NRF52 radio backend"
+    default NRF802154 if HAS_RADIO_NRF802154
+    default NRFBLE if HAS_RADIO_NRFBLE
+    default NRFMIN if HAS_RADIO_NRFMIN
+
+config NRF802154
+    bool "nrf802154"
+    imply MODULE_NRF802154
+
+config NRFBLE
+    bool "nrfble"
+    imply MODULE_NRFBLE
+
+config NRFMIN
+    bool "nrfmin"
+    imply MODULE_NRFMIN
```

Comment:
Perhaps we can prefix them with `NRF52_RADIO_`?

Reply:
Why do we need the indirection of having the choices `imply`ing modules? Can't we have the modules directly here? That way when dependencies are not met only available options show up.

Reply:
I wrongfully though that select would force the depency, not sure why..

Reply:
I'm not proposing to `select` them, just defining them here as options, instead of having intermediate symbols. If dependencies are not met they simply don't show up as a possible option.